### PR TITLE
CRM-Private Exchange Updates

### PR DESCRIPTION
**Pull request by nsSlalom**

### CODE REVIEW

**Comment by nsSlalom** [comment]
After this is merged, the validation rule "Three_Medical_Products_Required_PEBD" can then be deleted since it will not be in use for the Case object

File: src/classes/OpportunityPlaybookController.cls [hunk 1]
```diff
@@ -603,10 +603,11 @@ public with sharing class OpportunityPlaybookController
             if(caseRec.RecordTypeId == Schema.SObjectType.Case.getRecordTypeInfosByName().get( 'Proposal' ).getRecordTypeId())
                 caseRec.Bypass_Validation__c = TRUE;
             if (caseRec.RecordTypeId == Schema.SObjectType.Case.getRecordTypeInfosByName().get( System.label.Custom_Benefit_Service).getRecordTypeId()
-                    || caseRec.RecordTypeId == Schema.SObjectType.Case.getRecordTypeInfosByName().get( 'Mid-Market Underwriting' ).getRecordTypeId()
-                    || caseRec.RecordTypeId == Schema.SObjectType.Case.getRecordTypeInfosByName().get( Label.Case_Private_Exchange_Blue_Directions ).getRecordTypeId()
-                    || caseRec.RecordTypeId == Schema.SObjectType.Case.getRecordTypeInfosByName().get( Label.Case_Private_Exchange_External ).getRecordTypeId())
+                    || caseRec.RecordTypeId == Schema.SObjectType.Case.getRecordTypeInfosByName().get( 'Mid-Market Underwriting' ).getRecordTypeId())
```
Comment (rbonslater) [inline src/classes/OpportunityPlaybookController.cls]: @nsSlalom is there a way that we can get all of these hard coded strings into a constants utility class (e.g. CaseConstants or OpportunityConstants)?  Or for the sake of time, we could just define them at the top of the class (public static final String MID_MARKET_UNDERWRITING).

Comment (nsSlalom) [inline src/classes/OpportunityPlaybookController.cls]: Would custom labels work for this?

Comment (rbonslater) [inline src/classes/OpportunityPlaybookController.cls]: Yes, custom labels would be the best approach due to multi-lingual translations.

**Pull request merged**